src/app/_components/PageHeader.tsx

```diff
@@ -8,25 +8,84 @@ import {
   DescriptionText,
 } from '@/components/DescriptionText'
 import { Heading } from '@/components/Heading'
-import { Meta } from '@/components/Meta'
+import { Meta, type MetaDataType } from '@/components/Meta'
+import { NextStaticImage } from '@/components/NextStaticImage'
 import { SectionDivider } from '@/components/SectionDivider'
 
 import { type CTAProps } from '@/types/sharedProps/ctaType'
-import { type ImageProps } from '@/types/sharedProps/imageType'
+import {
+  type StaticImageProps,
+  type ImageProps,
+} from '@/types/sharedProps/imageType'
 
 type TitleProps = {
   children: string
 }
 
-type PageHeaderProps = {
+type SharedProps = {
   title: TitleProps['children']
   description?: DescriptionTextType
-  cta?: CTAProps
-  secondaryCta?: CTAProps
-  image?: ImageProps
-  metaData?: Array<string | null | undefined>
+  cta: CTAProps | [CTAProps, CTAProps?]
+  metaData?: MetaDataType
   isFeatured?: boolean
-  containImageSize?: boolean
+}
+
+type DynamicImageProps = Partial<ImageProps> & {
+  fallback: StaticImageProps
+  objectFit?: 'contain' | 'cover'
+}
```

Comment:
Should this live under `imageTypes`?

Reply:
`DynamicImageProps` now lives alongside `DynamicImage`, same for `StaticImageProps`.

I kept `ImageProps`, and reverted it to its original version, because it's still used by entry types (`BlogPostData` and the like), and all content entries use `url` for images. 

```ts
export type ImageProps = {
  url: string
  alt: string
}
```